README.md

```diff
@@ -19,15 +21,108 @@ We are a member of the [.NET Foundation](https://www.dotnetfoundation.org/about)
 
 ![Polly logo](https://raw.github.com/App-vNext/Polly/main/Polly-Logo.png)
 
-## Installing via the .NET SDK
+---
+
+<!-- This TOC can be maintained using VS Code's 'Markdown All in One' extension -->
```

Comment:
You could add this to the recommended Visual Studio Code extensions:

https://github.com/App-vNext/Polly/blob/330169f5b1be02830eac65fa82ca6f8534e7b830/.vscode/extensions.json#L2-L6